lua/telescope/builtin/init.lua

```diff
@@ -60,6 +60,17 @@ end
 
 local builtin = {}
 
+-- Ref: https://github.com/tjdevries/lazy.nvim
+local function require_on_exported_call(builtin_module)
```

Comment:
can you change `builtin_module` to just `mod`. It's not specific to `buitin` type modules.

After that, we can merge. Thanks!